cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/TestCommand.java

```diff
@@ -159,6 +159,9 @@ public TestCommand(Path userDir, PrintStream outStream, PrintStream errStream, b
     @CommandLine.Option(names = "--old-parser", description = "Enable old parser.", hidden = true)
     private boolean useOldParser;
 
+    @CommandLine.Option(names = "--rerun-failedTests", description = "Rerun failed tests.")
```

Comment:
Shall we rename the flag to `--rerun-failed`?

Reply:
Ack. Will change to `--rerun-failed`